content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -474,6 +474,56 @@ can lose track of volume labels after restart. This deficiency has been fixed
 in Kubernetes 1.26.
 {{< /note >}}
 
+## Changing the permissions of your's '/proc'
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+By default, containers have multiple paths that are both
+[masked](https://github.com/opencontainers/runtime-spec/blob/f66aad47309/config-linux.md#masked-paths)
+and read only. The result of this is the container has these paths present
+inside the container's mount namespace, and they can function similarly to if
+the container was an isolated host, but the container process cannot write to
+them. A list of the paths are found in the [Kubernetes source
+code](https://github.com/kubernetes/kubernetes/blob/964529b227/pkg/securitycontext/util.go#L193-L211):
+
+- Masked Paths:
+  - `/proc/asound`
+  - `/proc/acpi`
+  - `/proc/kcore`
+  - `/proc/keys`
+  - `/proc/latency_stats`
+  - `/proc/timer_list`
+  - `/proc/timer_stats`
+  - `/proc/sched_debug`
+  - `/proc/scsi`
+  - `/sys/firmware`
+
+- Read Only Paths:
+  - `/proc/bus`
+  - `/proc/fs`
+  - `/proc/irq`
+  - `/proc/sys`
+  - `/proc/sysrq-trigger`
+
+However, there are some cases where a user may wish their container did not
+mask these paths. The most common case is users trying to run containers within
+Kubernetes containers.
+
+The `securityContext` field `procMount` allows a user to request a container's `/proc`
+be `Unmasked`, or be mounted as read-write by the container process.
+
+```yaml
+...
+securityContext:
+  procMount: Unmasked
+```
+
+{{ < note > }}
+As of Kubernetes 1.30, this field requires the `spec.hostUsers` value in the pod
+spec to be `false`. In other words: a container that wishes to have an Unmasked
+`/proc` must also be in a [user
+namespace](/docs/concepts/workloads/pods/user-namespaces.md).
```

Comment:
```suggestion
{{ < note > }}
In Kubernetes {{< skew currentVersion >}}, this field requires the `spec.hostUsers` value in the pod
spec to be `false`. In other words: a container that wishes to have an Unmasked
`/proc` or unmasked `/sys` must also be in a
[user namespace](/docs/concepts/workloads/pods/user-namespaces/).

Kubernetes v1.12 to v1.29 did not enforce that requirement.
{{< /note >}}
```

If I want to map `/sys/firmware` into a (privileged) Pod, and I'm using Kubernetes v1.30, how do I do that?

Reply:
if a pod is privileged then no paths are masked

Reply:
OK, how about
```suggestion
{{ < note > }}
Setting `procMount` to Unmasked requires the `spec.hostUsers` value in the pod
spec to be `false`. In other words: a container that wishes to have an Unmasked
`/proc` or unmasked `/sys` must also be in a
[user namespace](/docs/concepts/workloads/pods/user-namespaces/).

Kubernetes v1.12 to v1.29 did not enforce that requirement.
{{< /note >}}
```
?

Reply:
Looking at the diff, it looks like this change was made; this can be resolved.